.github/workflows/flutter_integration.yaml

```diff
@@ -39,17 +39,17 @@ jobs:
   android:
     strategy:
       matrix:
-        api-level: [21, 29]
+        api-level: [24, 29]
```

Comment:
21 always fails with `java.lang.NoSuchMethodError: No virtual method setEndpointIdentificationAlgorithm` for a long time, it looks like we updated some library earlier that no longer support android 21